src/components/WifiCard.js

```diff
@@ -102,7 +102,7 @@ export const WifiCard = (props) => {
 
         <Pane width={'100%'}>
           <TextareaField
-            id="ssid"
+            id={'ssid' + (props.keyid === undefined ? '' : props.keyid)}
```

Comment:
Do this logic once outside of the return and then reuse it everywhere, maybe something like:

```jsx
const keyid = props.keyid || '';
const suffixKeyID = (prefix) => `${prefix}-${keyid}`

...

return (
  <textarea>
    id={suffixKeyID('ssid')}
     ...
  </textarea>
  <otherthing>
     id={suffixKeyID('eapmethod')}
     ...
  </otherthing>
)
```